example-mojaloop-backend/values.yaml

```diff
@@ -214,43 +105,42 @@ account-lookup-mysql:
     customPasswordFiles: {}
-  primary:
-    persistence:
-      enabled: false
-
-    ## @param secondary.affinity Affinity for MySQL secondary pods assignment
-    ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
-    ## Note: podAffinityPreset, podAntiAffinityPreset, and  nodeAffinityPreset will be ignored when it's set
-    ##
-    affinity: {}
-
-    ## @param secondary.nodeSelector Node labels for MySQL secondary pods assignment
-    ## ref: https://kubernetes.io/docs/user-guide/node-selection/
-    ##
-    # nodeSelector: {}
-    
-    ## Example
-    # nodeSelector:
-    #     "node-role.mojaloop.io": als
-
-    ## @param secondary.tolerations Tolerations for MySQL secondary pods assignment
-    ## ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
-    ##
-    # tolerations: []
-    
-    ## Example
-    # tolerations:
-    #   - key: "node-role.mojaloop.io"
-    #     operator: "Equal"
-    #     value: als
-    #     effect: "NoSchedule"
-
-  secondary:
-    persistence:
-      enabled: false
+  ## @param initdbScripts Dictionary of initdb scripts
+  ## Specify dictionary of scripts to be run at first boot
+  initdbScripts:
+    # This script enables legacy authentication for MySQL v8. NodeJS MySQL Client currently does not support authentication plugins, reference: https://github.com/mysqljs/mysql/pull/2233
+    # Note : also initialises mysql DBs to support thirdparty charts
+    accountLookupInit.sql: |-
+      DROP DATABASE IF EXISTS account_lookup;
+      CREATE DATABASE account_lookup;
+      DROP USER IF EXISTS account_lookup@'%';
+      CREATE USER 'account_lookup'@'%' IDENTIFIED WITH mysql_native_password BY 'HY#Mz%_z';
+      GRANT ALL PRIVILEGES ON account_lookup.* TO 'account_lookup'@'%';
+      FLUSH PRIVILEGES;
+    centralLedgerInit.sql: |-
+      DROP DATABASE IF EXISTS central_ledger;
+      CREATE DATABASE central_ledger;
+      DROP USER IF EXISTS central_ledger@'%';
+      CREATE USER 'central_ledger'@'%' IDENTIFIED WITH mysql_native_password BY 'HY#Mz%_z';
+      GRANT ALL PRIVILEGES ON central_ledger.* TO 'central_ledger'@'%';
+      FLUSH PRIVILEGES;  
+    consentOracleInit.sql: |-
+      DROP DATABASE IF EXISTS consent_oracle;
+      CREATE DATABASE consent_oracle;
+      DROP USER IF EXISTS consent_oracle@'%';
+      CREATE USER 'consent_oracle'@'%' IDENTIFIED WITH mysql_native_password BY 'HY#Mz%_z';
+      GRANT ALL PRIVILEGES ON consent_oracle.* TO 'consent_oracle'@'%';
+      FLUSH PRIVILEGES;
+    authSvcInit.sql: |-
+      DROP DATABASE IF EXISTS `auth-svc`;
+      CREATE DATABASE `auth-svc`;
+      DROP USER IF EXISTS `auth-svc`@'%';
+      CREATE USER 'auth-svc'@'%' IDENTIFIED WITH mysql_native_password BY 'HY#Mz%_z';
+      GRANT ALL PRIVILEGES ON `auth-svc`.* TO 'auth-svc'@'%';
+      FLUSH PRIVILEGES;
 
 ## Reference: https://github.com/bitnami/charts/blob/main/bitnami/mongodb/values.yaml
-centralledger-obj:
+cl-mongodb:
```

Comment:
Perhaps we should follow the same approach as MySQL here?

i.e. have a single DB with multiple schemas?

I think we only need two at the moment for the following components:
1. Central-event-processor
2. Bulk Adapters

Reply:
@mdebarros : Yes indeed I think this is a good idea but I think I made the comment earlier that I think this requires config for the init containers and it might be good to do that subsequent to this PR even if we do it in the v15 rel

Reply:
Ok, lets chat about that in tomorrow's meeting.

Reply:
Created a story for this: 

Separate story for MongoDB consolidation --> https://github.com/mojaloop/project/issues/3104